Cargo.toml

```diff
@@ -16,7 +16,7 @@ rust-version = "1.71"
 
 [dependencies]
 libc = "0.2.150"
-mozjpeg-sys = { version = "2.0.6", default-features = false, features = ["unwinding"] }
+mozjpeg-sys = { git = "https://github.com/kornelski/mozjpeg-sys.git", branch = "main", default-features = false, features = ["unwinding"] }
```

Comment:
Please revert this. For local dev I recommend `[patch]` section.

Reply:
Done

Reply:
Thanks! Updated the pr.